src/suffix.rs

```diff
@@ -214,10 +231,22 @@ impl SuffixScheme for AppendTimestamp {
     ) -> io::Result<TimestampSuffix> {
         assert!(suffix.is_none());
         if suffix.is_none() {
-            let now = Local::now().format(self.format).to_string();
+            let mut now = Local::now();
+
+            match self.date_from {
+                DateFrom::DateYesterday => {
+                    now = now - Duration::days(1);
+                }
+                DateFrom::DateHourAgo => {
+                    now = now - Duration::hours(1);
```

Comment:
Will this cause a time drift on the boundaries of a day (Say, during leap seconds)?

Reply:
Hm, maybe. it looks like, that in python the always count with seconds: https://github.com/python/cpython/blob/7fce1063b6e5a366f8504e039a8ccdd6944625cd/Lib/logging/handlers.py#L232